.github/workflows/on-pr.yaml

```diff
@@ -54,6 +54,7 @@ jobs:
       - name: Build helm chart
         run: |
           sed -i 's#registry/local/kai-scheduler#${{ env.DOCKER_REGISTRY }}#' deployments/kai-scheduler/values.yaml
+          sed -i "s/appVersion.*/appVersion: $PACKAGE_VERSION/" deployments/kai-scheduler/Chart.yaml
```

Comment:
You dont need it.
Helm package command get a flag "--app-version" to override the app version

Reply:
I actually looked for one and didn't find one.. where do you see it?

Reply:
![image](https://github.com/user-attachments/assets/fe01cb43-9aa2-43ee-bd78-fb3016456312)